resources/css/sharebetween.less

```diff
@@ -0,0 +1,15 @@
+.panel.sharebetween-wall-entry {
+  background: #fef8de;
```

Comment:
@yurabakhtin Can you please use 10% warning color, instead of hardcoded yellow.

Reply:
@luke- It seems you mean to use the CSS variable `--warning`, because no reason to use LESS variable `@warning` because we cannot get it from current theme, it is why we created some time ago the CSS variables.
I.e. the CSS code `background: #fef8de;` should be changed to `background: var(--warning);` and then apply some function to make the color 10%.
I found only a way to use `background: rgba(var(--warning), 0.1);`, but it doesn't work with HEX color values only RGB, e.g. if we will implement duplicates of the colors like `--warning-rgb: 255, 193, 7;` and it will looks like this:

![rgba](https://user-images.githubusercontent.com/6995524/225303782-f508bf47-8194-41d8-b15b-5325432fd05c.png)

But it is not same color what we have currently because it is transparented so it is mixed with grey background, I think it can be fixed if we will use an additional div with white background color, but firstly we should decide how to convert the HEX color to RGB, either convert them manually and keep their duplicates with suffixes `-rgb` or find... but I couldn't find
a CSS function that convert HEX to RGB.

---

Also I tried to use a color var `--background-color-warning: #fffbf7` but it is more lighter than `#fef8de`:

![--background-color-warning](https://user-images.githubusercontent.com/6995524/225305223-8359a620-ab03-4b8c-8a29-892ffefad9b8.png)

or `--border-color-warning: #fdd198`:

![--border-color-warning](https://user-images.githubusercontent.com/6995524/225305934-615742b1-a2fb-4151-a619-a1300a49d916.png)

---

Please note in core we use for current comment the color in LESS `fadeout(@warning, 75%);`, in CSS it is `rgba(255,193,7,0.25);`, and it looks like this:

![comment](https://user-images.githubusercontent.com/6995524/225307273-0eeeecdd-7047-458f-8602-0ec0502e278f.png)

but we see such light yellow color only because the back element has bg color as light grey, and if it was changed then the current comment color will be changed, for example:

![comment-2](https://user-images.githubusercontent.com/6995524/225307676-9391854b-0ded-455b-83bc-53d5fb4e04f1.png)

Maybe we should introduce some main color with name `@background-color-highlight` with value `#fef8de` without alpha channel, so then it looks same not depending on the back color:

![comment-3](https://user-images.githubusercontent.com/6995524/225308353-a98078c7-639a-419b-b1ce-5e17b6dacd88.png)

If we need to make new highlight color depending on the `@warning` colors we might implement new theme variable like this:

```css
@background-color-highlight: lighten(desaturate(spin(@warning, 3.7500), 5.8824), 41.9608);
--background-color-highlight: @background-color-highlight;
```

So then we can use it in other CSS files of external modules like `background: var(--background-color-highlight)`, where result will be `background: #fef8de`.

Reply:
@yurabakhtin Thanks for the explanations. I just discussed this with Lea and let's do this as suggested and use the existing `background-color-highlight` var. Instead of `@warning` lets use `@info` but make it double as light as it is now.     

New description text:
```
// Informative highlighting color
// (e.g. Comment Permalink, Shared Item, Calendar CurDay, Wiki Active Page Nav, Mail Module Speech Bubble)
@background-color-highlight: lighten(desaturate(spin(@info, 3.7500), 5.8824), 41.9608);
```

![image](https://user-images.githubusercontent.com/4736168/225353817-114f3db9-921e-426f-89c4-56a278801849.png)
![image](https://user-images.githubusercontent.com/4736168/225353901-1fe53c3f-3b8e-427c-b16e-c40b8f7f6950.png)
![image](https://user-images.githubusercontent.com/4736168/225354044-cfe5bec8-8555-48ac-a403-546d4fa2c86c.png)
![image](https://user-images.githubusercontent.com/4736168/225380370-d798084c-3217-4857-a6c6-ccc106714406.png)

For the Wiki we may need an additional `background-color-highlight-soft` variable for the child page highlighting in the nav.

Reply:
@luke- Core PR https://github.com/humhub/humhub/pull/6173, now it looks like this for current comment:

![info-bg](https://user-images.githubusercontent.com/6995524/225525560-78186afe-59d3-4aa0-a8aa-e0f2d6e6bbfc.png)

so even if a back element has another color it will looks same because no opacity effect:

![another-bg](https://user-images.githubusercontent.com/6995524/225525670-77e0f587-b174-48a3-8489-4421a549d492.png)

---

Sharebetween module https://github.com/humhub/sharebetween/pull/22/commits/cfe3baa5519b934dbcd2e3ad0aa0b40168bdcab9:

![shared-bg-info-color](https://user-images.githubusercontent.com/6995524/225525851-f8acb3a2-772e-4e28-b354-0639baa6d915.png)

---

Changes for other modules I will do and write below:

Reply:
@luke- @Eladnarlea Updates for wiki module https://github.com/humhub/wiki/pull/249/commits/6a8393abfdffca3492549562c58eeb4668e98230:

![wiki-colors](https://user-images.githubusercontent.com/6995524/225547204-0ad4877e-d90f-4b8f-a473-f97902515d2f.png)

`@background-color-highlight: lighten(desaturate(spin(@info, 4), 6), 42);` => `#bbe5ef`
`@background-color-highlight-soft: lighten(desaturate(spin(@info, 6), 16), 55);` => `#f2f9fb`

Reply:
@Eladnarlea @luke- Mail module PR https://github.com/humhub/mail/pull/314:

![mail-theme-var](https://user-images.githubusercontent.com/6995524/225550187-a8811b51-39a2-464b-a66b-598bc1388d4c.png)

Reply:
@Eladnarlea @luke- If new color `--background-color-highlight: `#bbe5ef` looks darker I can sugget another formula:

- `lighten(desaturate(spin(@info, 2), 17), 49)`

![mail-color2](https://user-images.githubusercontent.com/6995524/225552547-9ad2fb74-a79b-46a7-910b-13187bb078e9.png)

![wiki-color2](https://user-images.githubusercontent.com/6995524/225552553-f72ec3cd-dc6d-47e9-82c1-a8ebf56f86d4.png)

(currently the color is darken because I used a formula that was used to convert the `@warning` color to `#fef8de`, but if we switched to `@info` we can modify the formula to make it more correct what we want to see for default colors).

Reply:
I like this solution better :) thanks @yurabakhtin 

> formula:
> 
> * `lighten(desaturate(spin(@info, 2), 17), 49)`

Reply:
@Eladnarlea @luke- The `lighten(desaturate(spin(@info, 2), 17), 49)` => `#daf0f3` has been done in commit https://github.com/humhub/humhub/pull/6175/commits/73d87eebfe8803c045bcc786dade8c7ea37dc348.